### PR TITLE
Display the address with the card

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open your account page
- scroll down to your card
- you should see the address next to it

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7086/

File: jujugui/static/gui/src/app/components/account/payment-methods/card/card.js [hunk 1]
```diff
@@ -47,34 +47,43 @@ YUI.add('account-payment-method-card', function() {
...
+                  <div className="account__payment-card-brand">
+                    {card.brand}
+                  </div>
                 </div>
               </div>
             </div>
           </div>
+          <div className="account__payment-card-info">
+            <h4>Card address</h4>
+            <p>{address.line1} {address.line2}</p>
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/account/payment-methods/card/card.js]: You sure we want these sidebyside? They can get quite long...

Comment (huwshimi) [inline jujugui/static/gui/src/app/components/account/payment-methods/card/card.js]: yeah, probably not, not really sure how to display this stuff, but I'll update that at least.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7098/

**Comment by huwshimi** [comment]
Thanks @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**